api/v1beta1/openstackdataplaneservice_webhook.go

```diff
@@ -97,9 +187,31 @@ func (r *OpenStackDataPlaneService) ValidateUpdate(original runtime.Object) (adm
 }
 
 func (r *OpenStackDataPlaneServiceSpec) ValidateUpdate() field.ErrorList {
-	// TODO(user): fill in your validation logic upon object creation.
+	var errors field.ErrorList
+
+	// Validate issuers if present
+	if label, err := r.ValidateIssuer(); err != nil {
+		openstackdataplaneservicelog.Error(
+			err, "Error validating OpenStackDataPlaneService issuer",
+			"issuer", label)
+		errors = append(errors, field.Invalid(
+			field.NewPath("Spec").Child("TLSCert").Child("Issuer"),
+			label,
+			fmt.Sprintf("Error getting issuer with label %s: %s", label, err)))
+	}
 
-	return field.ErrorList{}
+	// Validate cacerts if present
+	if secretName, err := r.ValidateCACerts(); err != nil {
+		openstackdataplaneservicelog.Error(
+			err, "Error validating OpenStackDataPlaneService caCerts secret",
+			"secretName", secretName)
+		errors = append(errors, field.Invalid(
+			field.NewPath("Spec").Child("CACerts"),
```

Comment:
We should keep the `Spec` in lower case for sake of consistency. We didn't really do that before, but from now on we should.